platform/research/planning/mobile-research-guidelines.md

```diff
@@ -0,0 +1,87 @@
+
+# Mobile Research on VA.gov: Version 1
+
+## Planning research
+
+- Plan to have the participants complete the tasks using their own phone, as opposed to using an emulator on the desktop or shrinking the desktop browser down
+- While we primarily encourage testing on mobile devices, testing on tablets is also acceptable
+- Invision, UXPin, and staging are viable options for testing mobile flows on VA.gov
+- To bring more context into the situation, consider starting the prototype from what looks like a phone’s home screen (IOS or Android)  rather than from your design’s start page already open. (Make sure the first hotspot on that screen targets the phone’s browser icon - Safari or Chrome.)
+[Insert example screenshot]
+- Do pilot tests with team members with at least an iPhone and an Android phone
+- Prioritize a smaller amount of tasks (2-3) for the user test in case the initial phone and screen sharing setup takes longer than expected. Plan a few more tasks that are lower priority that you can get to if you have time.
+  - Alternatively, build a buffer into how long you say the session will take in planning/recruiting docs. E.g. we think it'll be 45 min, so schedule 1 hour and set expectation w/ participants that it'll be 45-60 min
+- Decide which method of mobile usability testing you'll use. Read more about that next.
+
+### Selecting a method
+There are two primary methods that are used for conducting mobile usability testing. Read about each one and their pros and cons.
+
+#### Laptop hugging method
+
+The “laptop hugging” method means that the user joins the call on their laptop and shows their mobile screen via the webcam. When the initial introduction to the session is done, the moderator asks the participant to turn their laptop 180 degrees so that the screen is looking away from the user. When the participant now places their hands around their computer and takes their mobile in their hands, it will be visible on the webcam.
+
+This photo shows what it looks like:
+![Image of a person demonstrating the laptop hugging method](https://miro.medium.com/max/1400/0*OUjnQaSUSbjutgPr)
+
+
+[Source, and more about the laptop hugging method](https://medium.com/@beparticular/were-still-hugging-our-laptops-8c7f22ed800e)
+
+Pros | Cons
+--------- | ----------
+Ability to see the full movement of the fingers | People without laptops or working web cam can’t use this method
+Relatively easy instruction and setup | The quality of the webcam could be blurry
+
+#### Screensharing method
+The screensharing method requires the participant to download the Zoom app on their phone and share their screen as they complete the tasks of the usability study. 
+
+Pros | Cons
+--------- | ----------
+People without laptops or webcams can use this method | Moderators can not see hand movements
+x | Could be difficult for non-tech savvy people (there might have to be some reconfiguration in the settings section of the phone)
```

Comment:
another "pro" is that Perigean has already helped folks run studies in this way, so it's been socialized with them already. That said, we could work with them on the laptop hugging method